package.json

```diff
@@ -39,43 +39,37 @@
     "lodash": "^4.17.20",
     "luxon": "^1.25.0",
     "notistack": "^1.0.3",
-    "postcss-focus-visible": "^5.0.0",
     "react": "16.13.1",
     "react-chartjs-2": "^2.11.1",
     "react-dom": "16.13.1",
     "react-input-mask": "^2.0.4",
     "react-modal": "^3.12.1",
     "react-quill": "2.0.0-beta.2",
     "react-text-mask": "^5.4.3",
+    "tailwindcss": "^2.2.4",
     "yup": "^0.32.8"
```

Comment:
I think all our dependencies can be `devDependencies`, right? Especially since we've moved all of them to `peerDependencies`.

Reply:
probably, but I don't think it matters, I'd like to keep these ones separate because they are the ones used for the actual components. just an easier way to see what should match with the peer deps